recipes/crispritz/meta.yaml

```diff
@@ -1,22 +1,23 @@
-{% set version = "1.0.5" %}
+{% set version = "1.0.6" %}
 
 package:
   name: crispritz
   version: {{ version }}
 
 source:
   url: https://github.com/InfOmics/CRISPRitz/archive/v{{ version }}.tar.gz
-  sha256: ec616f6f20c498dc6b84232c52daac4462640fd0e3630b1e267140e95d3e2f63
-
+  sha256: 2030530c40a713e0e387b80b4283bcb6aa12c942ecde58ec45cfd0e5c8285071
+  
 build:
-  number: 0
-  skip: True # [osx]
+  number: 1
 
 requirements:
   build:
-    - {{compiler('cxx')}} =5.4
+    - {{compiler('cxx')}} =5.4 # [not osx]
```

Comment:
This is very very likely to break in all manner of unexpected ways. The software does not compile with gcc 7.2 at all? Can't be patched?

Reply:
Hi,
I'm sorry but I tried every version except the 5.4 for gcc and no one can run properly.
I discovered it's a problem with the dynamic bitset variable in the boost library, but I have not found a solution except using the mentioned gcc version.
Another question, I'm having a problem with openmp on the clang compiler for macOS X, I included openmp in the recipe, but the error remains, there's something I'm missing?

Thanks a lot.

Reply:
I'd have to look at it. Boost should compile fine. It usually is just some warnings that became errors as GCC becomes more stringent. I'd have to compile myself to see. Most of the time, some small patches suffice to fix those compilation issues.